<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindows</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>升级固件：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Open">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>打 开</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>本地IP：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_LocalIP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>自动升级</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_AutoControllerPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>控制器IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>控制器端口：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_AutoControllerIP">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Update_aotoGet">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>自动获取</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>手动升级</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_ManualControllerPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ManualControllerIP">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>控制器IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>控制器端口：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Update">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>升  级</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Firmware">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_O"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_D"/>
    <addaction name="action_A"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>选项(&amp;O)</string>
    </property>
    <addaction name="action_Log"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_O"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_O"/>
   <addaction name="action_Debug"/>
   <addaction name="separator"/>
   <addaction name="action_D"/>
  </widget>
  <action name="action_O">
   <property name="icon">
    <iconset resource="Res/res.qrc">
     <normaloff>:/Res/Open.ico</normaloff>:/Res/Open.ico</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_D">
   <property name="icon">
    <iconset resource="Res/res.qrc">
     <normaloff>:/Res/112.ico</normaloff>:/Res/112.ico</iconset>
   </property>
   <property name="text">
    <string>升级说明(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="Res/res.qrc">
     <normaloff>:/Res/About.ico</normaloff>:/Res/About.ico</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Debug">
   <property name="icon">
    <iconset resource="Res/res.qrc">
     <normaloff>:/Res/favicon.ico</normaloff>:/Res/favicon.ico</iconset>
   </property>
   <property name="text">
    <string>日志</string>
   </property>
  </action>
  <action name="action_Log">
   <property name="icon">
    <iconset resource="Res/res.qrc">
     <normaloff>:/Res/favicon.ico</normaloff>:/Res/favicon.ico</iconset>
   </property>
   <property name="text">
    <string>日志</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_Open</tabstop>
  <tabstop>comboBox_LocalIP</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_AutoControllerPort</tabstop>
  <tabstop>comboBox_AutoControllerIP</tabstop>
  <tabstop>pushButton_Update_aotoGet</tabstop>
  <tabstop>lineEdit_ManualControllerPort</tabstop>
  <tabstop>lineEdit_ManualControllerIP</tabstop>
  <tabstop>pushButton_Update</tabstop>
  <tabstop>lineEdit_Firmware</tabstop>
 </tabstops>
 <resources>
  <include location="Res/res.qrc"/>
 </resources>
 <connections/>
</ui>
